cirq/aqt/__init__.py

```diff
@@ -12,6 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cirq.aqt.aqt_sampler import AQTSampler, AQTSamplerLocalSimulator
-from cirq.aqt.aqt_device import AQTSimulator
+from cirq.aqt.aqt_sampler import (
+    AQTSampler,
+    AQTSamplerLocalSimulator,
+)
+
+from cirq.aqt.aqt_device import (
+    AQTSimulator,
+    AQTNoiseModel,
+    get_aqt_device,
+    get_aqt_default_noise_dict,
+    _get_aqt_device_op_string,
```

Comment:
Do you want to expose _get_aqt_device_op_string to users?  If so I would not make it private (remove '_')